chain/chain/src/flat_storage_resharder.rs

```diff
@@ -316,8 +328,12 @@ impl FlatStorageResharder {
             if self.controller.is_cancelled() {
                 return FlatStorageReshardingTaskStatus::Cancelled;
             }
+
+            // Sleep between batches in order to throttle resharding and leave some resource for the
+            // regular node operation.
+            std::thread::sleep(batch_delay);
         }
-        FlatStorageReshardingTaskStatus::Successful
+        FlatStorageReshardingTaskStatus::Successful { batches_done }
```

Comment:
very nit. maybe rename to num_batches_done? or num_batches, since the status is already successfulo.